modules/products/pycharm/src/main/kotlin/com/github/l34130/mise/pycharm/run/MisePythonCommandLineTargetEnvironmentProvider.kt

```diff
@@ -16,16 +17,17 @@ class MisePythonCommandLineTargetEnvironmentProvider : PythonCommandLineTargetEn
         pythonExecution: PythonExecution,
         runParams: PythonRunParams,
     ) {
-        if (runParams is AbstractPythonRunConfiguration<*>) {
-            val envs =
-                MiseHelper.getMiseEnvVarsOrNotify(
-                    configuration = runParams,
-                    workingDirectory = { runParams.workingDirectory },
-                )
```

Comment:
The previous prefers the Run configuration's options but the current is not.

I think it will be great if we use the new logic as a fallback of previous, or handle explicitly by the runParams' type.